subworkflows/local/align_cellrangermulti_idx.nf

```diff
@@ -0,0 +1,89 @@
+//
+// Include modules
+//
+include { CELLRANGER_MKGTF                  } from "../../modules/nf-core/cellranger/mkgtf/main.nf"
+include { CELLRANGER_MKREF                  } from "../../modules/nf-core/cellranger/mkref/main.nf"
+include { CELLRANGER_MKVDJREF               } from "../../modules/nf-core/cellranger/mkvdjref/main.nf"
+
+// Define workflow to subset and index a genome region fasta file
+workflow CELLRANGER_MULTI_REF {
+    take:
+        ch_fasta
+        ch_gtf
+        cellranger_gex_index
+        cellranger_vdj_index
+
+    main:
+        ch_versions    = Channel.empty()
+
+        //
+        // TODO: Include checkers for cellranger multi parameter combinations. For example, when VDJ data is given, require VDJ ref. If FFPE, require frna probe sets, etc.
+        //
+
+        // Assign other cellranger reference files
+        ch_gex_frna_probeset      = params.gex_frna_probe_set            ? file(params.gex_frna_probe_set)            : []
+        ch_gex_target_panel       = params.gex_target_panel              ? file(params.gex_target_panel)              : []
+        ch_gex_cmo_set            = params.gex_cmo_set                   ? file(params.gex_cmo_set)                   : []
+        ch_gex_barcodes           = params.gex_barcode_sample_assignment ? file(params.gex_barcode_sample_assignment) : []
+        ch_fb_reference           = params.fb_reference                  ? file(params.fb_reference)                  : []
+        ch_vdj_primer_index       = params.vdj_inner_enrichment_primers  ? file(params.vdj_inner_enrichment_primers)  : []
```

Comment:
I don't think all of them are needed here?

Reply:
no you are right ^^ There gone!